unity-ggjj/Assets/Scripts/NarrativeScript/MissingBackgroundTagException.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Linq;
+using Ink.Runtime;
+
+public class MissingBackgroundTagException : Exception
+{
+    public MissingBackgroundTagException(Story story) : base(CreateExceptionMessage(story)) { }
+
+    private static string CreateExceptionMessage(Story story)
+    {
+        var message = 
+            $"Could not find background tag near '{story.currentText.Replace("\n", "")}'\n" +
+            $"Move tag is missing a valid corresponding background tag in the format '#{InvestigationState.BACKGROUND_TAG_KEY}:[background name]'\n";
```

Comment:
This could be a bit less wordy and use the terminology from the [constants documentation](https://github.com/Studio-Lovelies/GG-JointJustice-Unity/blob/develop/docs/constants.md#sceneassetname):
```suggestion
            $"'#Move' choices also require a background tag: '#{InvestigationState.BACKGROUND_TAG_KEY}:[SceneAssetName]'\n";
```